istio-control/istio-discovery/values.yaml

```diff
@@ -124,4 +124,21 @@ telemetry:
   v2:
     # For Null VM case now. If enabled, will set disableMixerHttpReports to true and not define mixerReportServer
     # also enable metadata exchange and stats filter.
-    enabled: false
\ No newline at end of file
+    enabled: false
+    # Indicate if prometheus stats filter is enabled or not
+    prometheus:
+      enabled: false
+    # stackdriver filter settings.
+    stackdriver:
+      enabled: true
```

Comment:
Stackdriver should be disabled by default as well.

Reply:
I updated the stackdriver configs to be disabled, now then if we want to enable stackdriver filter then we need to do sth like this: 

istioctl manifest apply --set values.telemetry.enabled=true --set values.telemetry.v2.enabled=true --set values.telemetry.v2.stackdriver.enabled=true

A little bit lengthy but less confusing.